sites/friday/src/get-started/flask/deploy/configure.md

```diff
@@ -0,0 +1,241 @@
+```
+
+It will then ask for the name of your application. From there it should prompt you for services your project
+needs. Select each one and then hit Enter. For this demonstration, I only need `PostgreSQL`
+
+```shell
+
+                       (\_/)
+We’re almost done...  =(^.^)=
+
+Last but not least, unless you’re creating a static website, your project uses services. Let’s define them:
+
+Select all the services you are using:
+Use arrows to move, space to select, type to filter
+  [ ]  MariaDB
+  [ ]  MySQL
+> [x]  PostgreSQL
+  [ ]  Redis
+  [ ]  Redis Persistent
+  [ ]  Memcached
+  [ ]  OpenSearch
+```
+
+It will then generate a series of configuration files for you:
+
+```shell
+
+┌───────────────────────────────────────────────────┐
+│   CONGRATULATIONS!                                │
+│                                                   │
+│   We have created the following files for your:   │
+│     - .environment                                │
+│     - {{< vendor/configfile "app" >}}                         │
+│                                                   │
+│   We’re jumping for joy! ⍢                        │
+└───────────────────────────────────────────────────┘
+         │ /
+         │/
+         │
+  (\ /)
+  ( . .)
+  o (_(“)(“)
+
+You can now deploy your application to {{% vendor/name %}}!
+To do so, commit your files and deploy your application using the {{% vendor/name %}} CLI:
+  $ git add .
+  $ git commit -m 'Add {{% vendor/name %}} configuration files'
+  $ {{% vendor/cli %}} project:set-remote
+  $ {{% vendor/cli %}} push
+```
+
+Last, we need to add all of our generated files, from both Cookiecutter and the {{% vendor/name %}} CLI tool to our git
+repository:
+```shell
+$ git add .
+$ git commit -m "initial commit"
+```
+
+Before we can deploy our application, we'll need to create a new project on {{% vendor/name %}}. From the command line:
+```shell
+$ {{% vendor/cli %}} project:create
+```
+The CLI tool will now walk you through the creation of a project asking you for your organization, the project's
+title, the region where you want the application housed, and the branch name (use the same one we set
+earlier). For now, allow the cli tool to set {{% vendor/name %}} as your repository's remote, and then select `Y` to
+allow the tool to create the project. The {{% vendor/name %}} bot will begin the generation of your {{% vendor/name %}}
+project and once done, will report back the details of your project including the project's ID, and URL to where you can
+manage the project from the {{% vendor/name %}} web console. Don't worry if you forget any of this information; you can
+retrieve it later with:
+```shell
+$ {{% vendor/cli %}} project:info
+```
+
+And you can launch the web console for your project at any time by doing
+```shell
+$ {{% vendor/cli %}} web
+```
+
+
+Now that we have our {{% vendor/name %}} project created, our local project generated and associated with the
```

Comment:
I wonder if this spot should be the end of this page in the guide, and everything from this point to the end be moved into its own section?